src/test/java/org/folio/rest/impl/EntitiesCrudBasicsTest.java

```diff
@@ -124,7 +125,7 @@ static Stream<TestEntities> getTestEntitiesWithGetEndpoint() {
   }
 
   static Stream<TestEntities> getTestEntitiesWithGetEndpointWithoutGroup() {
-    return getTestEntitiesWithGetEndpoint().filter(e -> !e.equals(GROUP));
+    return getTestEntitiesWithGetEndpoint().filter(e -> !e.equals(FISCAL_YEAR) && !e.equals(GROUP));
```

Comment:
Please use ObjectUtils.notEqual static method. It will help to improve readability.

Reply:
fixed